samples/react-command-convert-to-pdf/gulpfile.js

```diff
@@ -34,4 +42,7 @@ build.rig.getTasks = function () {
   return result;
 };
 
-build.initialize(gulp);
\ No newline at end of file
+
+require("./spfx-versioning")(build);
+build.tslintCmd.enabled = false;
```

Comment:
this is not a huge problem or a bug but seems to me a bit out of scope for the related issue. Our aim is to upgrade the project and fix it up if anything requires updates to make it work with SPFx 1.20. 
Here we are adding an additional file for auto update of version number. I say lets avoid including additional side changes and make our update and PR simple and clean 👍

Reply:
Thanks for comments, feedback and guidance!
I've never used the SPFX upgrade CLI, Didn't know it existed!! That's gonna help quite a bit!

I'll fix it right away and make a new pull request! :)

Reply:
no problem. in the related issue I mentioned two tools that might support you in the upgrade. 
You may use either CLI for Microsoft 365 using the `m365 spfx project upgrade` command 
or You may install the SPFx Toolkit VS Code extension and there is an upgrade action as well